src/filter.js

```diff
@@ -111,6 +122,9 @@ function addDependenciesOfTable(allTables, accTables, currentTable, excludeBackl
       if (!ignoreMissing) {
         console.warn("Linking to a missing table - ignoring!", ignoreMissing);
       }
+    } else if (filterBacklinks && tableId === excludedTableId) {
+      // do not care about first table
+      console.log("back link to first table", tableId);
```

Comment:
Don't really know if term back link is correct. A link doesn't really have a direction. It is only a back link from the filters starting point. Also is this log necessary? Doesn't this pollute the aggregation log?

Reply:
Log will be removed